requirements.txt

```diff
@@ -9,4 +9,4 @@ pytest-cov==1.8.1
 tox==1.8.1
 virtualenv==1.11.6
 intelhex
-
+pyudev
```

Comment:
It's for cycling the hub, but it should probably be moved out of requirements and into the script.